### PR TITLE
65650712 generic delete action for a resource

### PR DESCRIPTION
**Pull request by ke4**

### CODE REVIEW

File: lib/lims-api/core_resource.rb [hunk 1]
```diff
@@ -109,9 +109,19 @@ def deleter
           dedicated_action_for(action_class, :"#{model_name}_uuid" => uuid)
         else
           @context.with_session do |session|
-            # load the object
-            object(session)            
+            # load the object and mark it for deletion from the DB
+            session << object(session)
```
Comment (maxigit) [inline lib/lims-api/core_resource.rb:113]: Is this change really necessary ? object(session) add the object to the session.
Or is that needed if the object has been loaded from another session ?

Comment (ke4) [inline lib/lims-api/core_resource.rb:113]: Without this modification we just load the object into the session, but we won't manage its state.
So, when we would change the object state to deleted, then we won't call the function to delete it.
